lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -528,7 +528,47 @@ def test_cache_map_success(self, monkeypatch, tmp_path):
                 == "symlink"
             )
 
-    def test_cache_map_bad_dir_path(self, monkeypatch, tmp_path):
+    @pytest.mark.parametrize(
+        "file_path, expected_msg",
+        [
+            (
+                "/dir_name/subdir1/f11.txt",
+                "The directory path '/dir_name/subdir1/f11.txt' is an absolute path, we expect relative path to the root directory.",
+            ),
+            (
+                "dir_name/subdir1/subdir11/../f11.txt",
+                "File/directory 'children' in path dir_name/subdir1/subdir11/../f11.txt not found in cache.",
+            ),
```

Comment:
I don't think that this error message is helpful:  we should be objecting to `'..'`, not to `'children'`.